src/FunnelChart/FunnelChart.tsx

```diff
@@ -120,9 +146,10 @@ export const FunnelChart: FC<FunnelChartProps> = ({
       margins={margins}
       containerClassName={containerClassName}
       className={className}
+      onClick={(e: MouseEvent) => handleOnClick(e, getDatas({ chartHeight: height, chartWidth: width }).datas[0])}
```

Comment:
This is a little complex to be on one line - can we break it out for readability.

Reply:
While breaking this out I realized I wasn't using the correct `chartHeight` and `chartWidth` for the Autosize FunnelChart, so I made some changes to store the chart dimensions in state and use them here